### PR TITLE
Add Bandit scanning for security issues in Python code

### PR DESCRIPTION
**Pull request by anzhella-pankratova**

### CODE REVIEW

File: .bandit [hunk 1]
```diff
@@ -0,0 +1,399 @@
+
```
Comment (Wovchena) [inline .bandit:1]: You should exclude it in `.gitattributes` so we wouldn't pack the file.

File: ci/prepare-documentation.py [hunk 2]
```diff
@@ -42,7 +42,7 @@
 import sys
 import urllib.parse
 import urllib.request
-import xml.etree.ElementTree as ET
+import xml.etree.ElementTree as ET # nosec - disable B405:import-xml-etree check
```
Comment (anzhella-pankratova) [inline ci/prepare-documentation.py:45]: The recommendation here is to replace `xml.etree.ElementTree` with the equivalent `defusedxml` package. However, the  `defusedxml` doesn't contain `Element` and `SubElement` classes or equivalent one, therefore this replacement would require changes in `add_page` implementation. 
For now the implementation with `xml.etree` is left.

File: .bandit (hunk 1 above)
Comment (anzhella-pankratova) [reply .bandit:1]: Updated

File: ci/prepare-documentation.py (hunk 2 above)
Comment (Wovchena) [reply ci/prepare-documentation.py:45]: Ok

**Review by Wovchena — APPROVED** [review]
Thank you!

**Comment by Wovchena** [comment]
CC @IRDonch

**Comment by Wovchena** [comment]
Jenkins please retry a build

**Review by eaidova — APPROVED** [review]

**Pull request merged**